community/contributing/pr_workflow.rst

```diff
@@ -86,13 +95,17 @@ repository (in Git speak, the *origin remote*):
           not be typed.
 
 After a little while, you should have a ``godot`` directory in your current
-working directory. Move into it (``cd godot``), and we will set up a useful
-reference:
+working directory. Move into it using the cd command:
+
+::
+    
+    $ cd godot 
+
+and now we will set up a useful reference:
 
 ::
 
     $ git remote add upstream https://github.com/godotengine/godot
-    $ git fetch upstream
```

Comment:
Why remove that?

Reply:
As far as I'm aware the fetch command has nothing to do with setting up a reference to the original repository.

Reply:
Well it fetches the branches of that remote repository, so that you have them locally and can do e.g. `git reset --hard upstream/master` (you can't do a hard reset on remote branches, only local). But we don't show any use of local upstream branches, so I guess it's fine to remove it if it really bothers you :)